docs/operations/rule-configuration.md

```diff
@@ -23,212 +23,242 @@ title: "Retaining or automatically dropping data"
+Set the following properties:
+- `period`: a JSON object representing [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) periods. The period is from some time in the past to the future or to the current time, depending on the `includeFuture` flag.
+- `includeFuture`: a boolean flag to indicate whether the drop period includes the future. Defaults to `true`.
 
-The interval of a segment will be compared against the specified period. The period is from some time in the past to the future or to the current time, which depends on `includeFuture` is true or false. The rule matches if the period *contains* the interval. This drop rule always dropping recent data.
+Druid compares a segment's interval to the period you specify in the rule. The rule matches if the period contains the segment interval. This rule always drops recent data.
 
-### Period Drop Before Rule
+### Period drop before rule
 
-Period drop before rules are of the form:
+Period drop rules have type `dropBeforeByPeriod` and the following example API payload:
 
 ```json
 {
-  "type" : "dropBeforeByPeriod",
-  "period" : "P1M"
+  "type": "dropBeforeByPeriod",
+  "period": "P1M"
 }
 ```
 
-* `type` - this should always be "dropBeforeByPeriod"
-* `period` - A JSON Object representing ISO-8601 Periods
+Set the following property:
+- `period`: a JSON object representing [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) periods.
+
+Druid compares a segment's interval to the period you specify in the rule. The rule matches if the segment interval is before the specified period. 
 
-The interval of a segment will be compared against the specified period. The period is from some time in the past to the current time. The rule matches if the interval before the period. If you just want to retain recent data, you can use this rule to drop the old data that before a specified period and add a `loadForever` rule to follow it. Notes, `dropBeforeByPeriod + loadForever` is equivalent to `loadByPeriod(includeFuture = true) + dropForever`.
+If you only want to retain recent data, you can use this rule to drop old data before a specified period, and add a `loadForever` rule to retain the data that follows it. Note that the rule combination `dropBeforeByPeriod` + `loadForever` is equivalent to `loadByPeriod(includeFuture = true)` + `dropForever`.
 
-## Broadcast Rules
+## Broadcast rules
 
-Broadcast rules indicate that segments of a data source should be loaded by all servers of a cluster of the following types: historicals, brokers, tasks, and indexers.
+Broadcast rules instruct Druid to load segments of a data source in all brokers, historicals, tasks, and indexers in the cluster.
 
-Note that the broadcast segments are only directly queryable through the historicals, but they are currently loaded on other server types to support join queries.
+Note that the broadcast segments are only directly queryable through the historicals, but Druid loads them on other server types to support join queries.
```

Comment:
One may wonder why this would be needed.

> The Broadcast rule defines a "broadcast table", typically to assist with broadcast joins. A broadcast join allows you to lookup (i.e. join) values in the broadcast. Broadcast tables are used by the index table feature (provide link).